libraries/AP_Camera/AP_Camera_SoloGimbal.cpp

```diff
@@ -0,0 +1,89 @@
+#include "AP_Camera_SoloGimbal.h"
+#include <GCS_MAVLink/GCS.h>
+
+uint8_t AP_Camera_SoloGimbal::gopro_capture_mode;
```

Comment:
This should be an instance of `AP_Camera_SoloGimbal`.

Reply:
Unsure what you mean?